### DISCUSSION TITLE
no parameter to specify compartibility for java versions

### DISCUSSION THREAD

**Discussion by P1tt187** [renovatebot/renovate#25407] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.32.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I use Renovate to provide an internal docker container in my company.
I use the `java-version` datasource to determine the version.
Currently there is no option to specify comparitbilty parameters.

In my case i need binarys for linux x64.
This leads to renovate offering the update too early.

A good example is the following release page:
https://github.com/adoptium/temurin8-binaries/releases/tag/jdk8u392-b08

The first artefacts where uploaded on 18 October, the artifacts I need where available on 23 October.
So I got a pullrequest that cannot build for 5 days.

It would be nice append the parameters `architecture` and `os`. 
In this case renovate would offer the update only when it is available.

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
You have reported a problem, but what you have described is not a problem. Ask a question instead.
